<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Function reference • bears</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><meta property="og:image" content="/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-index">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">bears</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/bears.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/almeidasilvaf/bears/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index"><colgroup><col class="alias"><col class="title"></colgroup><tbody><tr><th colspan="2">
          <h2 id="all-functions">All functions <a href="#all-functions" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="create_dir_structure.html">create_dir_structure()</a></code> </p>
        </td>
        <td><p>Create a standard directory for bears</p></td>
      </tr><tr><td>
          <p><code><a href="create_sample_info.html">create_sample_info()</a></code> </p>
        </td>
        <td><p>Search the SRA database and create a data frame of sample metadata</p></td>
      </tr><tr><td>
          <p><code><a href="download_fastq.html">download_fastq()</a></code> </p>
        </td>
        <td><p>Download FASTQ files</p></td>
      </tr><tr><td>
          <p><code><a href="download_from_ena.html">download_from_ena()</a></code> </p>
        </td>
        <td><p>Download FASTQ files from ENA's FTP</p></td>
      </tr><tr><td>
          <p><code><a href="fastq_exists.html">fastq_exists()</a></code> </p>
        </td>
        <td><p>Check if FASTQ files were properly downloaded</p></td>
      </tr><tr><td>
          <p><code><a href="fastqc_is_installed.html">fastqc_is_installed()</a></code> </p>
        </td>
        <td><p>Check if FastQC is installed</p></td>
      </tr><tr><td>
          <p><code><a href="fastqc_table.html">fastqc_table</a></code> </p>
        </td>
        <td><p>FastQC summary table</p></td>
      </tr><tr><td>
          <p><code><a href="fcount.html">fcount()</a></code> </p>
        </td>
        <td><p>Count reads with featureCounts</p></td>
      </tr><tr><td>
          <p><code><a href="featureCounts2se.html">featureCounts2se()</a></code> </p>
        </td>
        <td><p>Create a SummarizedExperiment object from featureCounts output</p></td>
      </tr><tr><td>
          <p><code><a href="get_url_ena.html">get_url_ena()</a></code> </p>
        </td>
        <td><p>Get URL for each file in the ENA's FTP repository</p></td>
      </tr><tr><td>
          <p><code><a href="gff2bed.html">gff2bed()</a></code> </p>
        </td>
        <td><p>Convert GFF file to BED</p></td>
      </tr><tr><td>
          <p><code><a href="infer_strandedness.html">infer_strandedness()</a></code> </p>
        </td>
        <td><p>Infer library strandedness</p></td>
      </tr><tr><td>
          <p><code><a href="kallisto2se.html">kallisto2se()</a></code> </p>
        </td>
        <td><p>Create a SummarizedExperiment object from kallisto output</p></td>
      </tr><tr><td>
          <p><code><a href="kallisto_index.html">kallisto_index()</a></code> </p>
        </td>
        <td><p>Index the transcriptome for kallisto</p></td>
      </tr><tr><td>
          <p><code><a href="kallisto_is_installed.html">kallisto_is_installed()</a></code> </p>
        </td>
        <td><p>Check if kallisto is installed</p></td>
      </tr><tr><td>
          <p><code><a href="kallisto_quantify.html">kallisto_quantify()</a></code> </p>
        </td>
        <td><p>Quantify expression with kallisto</p></td>
      </tr><tr><td>
          <p><code><a href="mapping_pass.html">mapping_pass()</a></code> </p>
        </td>
        <td><p>Keep only samples that passed minimum requirements in STAR alignment</p></td>
      </tr><tr><td>
          <p><code><a href="mapping_qc.html">mapping_qc</a></code> </p>
        </td>
        <td><p>Summary statistics of STAR mapping QC</p></td>
      </tr><tr><td>
          <p><code><a href="multiqc.html">multiqc()</a></code> </p>
        </td>
        <td><p>Run MultiQC to get a summary of QC results</p></td>
      </tr><tr><td>
          <p><code><a href="multiqc_is_installed.html">multiqc_is_installed()</a></code> </p>
        </td>
        <td><p>Check if multiqc is installed</p></td>
      </tr><tr><td>
          <p><code><a href="remove_rrna.html">remove_rrna()</a></code> </p>
        </td>
        <td><p>Remove rRNA sequences from .fastq files with SortMeRNA</p></td>
      </tr><tr><td>
          <p><code><a href="rseqc_is_installed.html">rseqc_is_installed()</a></code> </p>
        </td>
        <td><p>Check if RSeQC is installed</p></td>
      </tr><tr><td>
          <p><code><a href="run_fastqc.html">run_fastqc()</a></code> </p>
        </td>
        <td><p>Run FastQC on .fastq files</p></td>
      </tr><tr><td>
          <p><code><a href="salmon2se.html">salmon2se()</a></code> </p>
        </td>
        <td><p>Create a SummarizedExperiment object from salmon output</p></td>
      </tr><tr><td>
          <p><code><a href="salmon_index.html">salmon_index()</a></code> </p>
        </td>
        <td><p>Index the transcriptome for salmon</p></td>
      </tr><tr><td>
          <p><code><a href="salmon_is_installed.html">salmon_is_installed()</a></code> </p>
        </td>
        <td><p>Check if salmon is installed</p></td>
      </tr><tr><td>
          <p><code><a href="salmon_quantify.html">salmon_quantify()</a></code> </p>
        </td>
        <td><p>Quantify expression with salmon</p></td>
      </tr><tr><td>
          <p><code><a href="sample_info.html">sample_info</a></code> </p>
        </td>
        <td><p>Sample metadata</p></td>
      </tr><tr><td>
          <p><code><a href="sortmerna_is_installed.html">sortmerna_is_installed()</a></code> </p>
        </td>
        <td><p>Check if SortMeRNA is installed</p></td>
      </tr><tr><td>
          <p><code><a href="sratoolkit_is_installed.html">sratoolkit_is_installed()</a></code> </p>
        </td>
        <td><p>Check if SRAToolkit is installed</p></td>
      </tr><tr><td>
          <p><code><a href="star_align.html">star_align()</a></code> </p>
        </td>
        <td><p>Align reads to a reference genome using STAR</p></td>
      </tr><tr><td>
          <p><code><a href="star_genome_index.html">star_genome_index()</a></code> </p>
        </td>
        <td><p>Index genome for STAR alignment</p></td>
      </tr><tr><td>
          <p><code><a href="star_is_installed.html">star_is_installed()</a></code> </p>
        </td>
        <td><p>Check if STAR is installed</p></td>
      </tr><tr><td>
          <p><code><a href="stringtie2se.html">stringtie2se()</a></code> </p>
        </td>
        <td><p>Create a SummarizedExperiment object from featureCounts output</p></td>
      </tr><tr><td>
          <p><code><a href="stringtie_assemble.html">stringtie_assemble()</a></code> </p>
        </td>
        <td><p>Assemble transcripts with StringTie</p></td>
      </tr><tr><td>
          <p><code><a href="stringtie_is_installed.html">stringtie_is_installed()</a></code> </p>
        </td>
        <td><p>Check if StringTie is installed</p></td>
      </tr><tr><td>
          <p><code><a href="stringtie_quantify.html">stringtie_quantify()</a></code> </p>
        </td>
        <td><p>Quantify expression in TPM with StringTie</p></td>
      </tr><tr><td>
          <p><code><a href="subread_is_installed.html">subread_is_installed()</a></code> </p>
        </td>
        <td><p>Check if subread is installed</p></td>
      </tr><tr><td>
          <p><code><a href="taco_is_installed.html">taco_is_installed()</a></code> </p>
        </td>
        <td><p>Check if TACO is installed</p></td>
      </tr><tr><td>
          <p><code><a href="taco_merge.html">taco_merge()</a></code> </p>
        </td>
        <td><p>Merge assembled transcripts with TACO</p></td>
      </tr><tr><td>
          <p><code><a href="translate_strandedness.html">translate_strandedness()</a></code> </p>
        </td>
        <td><p>Translate library orientation terminology for each program</p></td>
      </tr><tr><td>
          <p><code><a href="trim_reads.html">trim_reads()</a></code> </p>
        </td>
        <td><p>Run Trimmomatic on reads that failed on FastQC check</p></td>
      </tr><tr><td>
          <p><code><a href="trimmomatic_is_installed.html">trimmomatic_is_installed()</a></code> </p>
        </td>
        <td><p>Check if Trimmomatic is installed</p></td>
      </tr><tr><td>
          <p><code><a href="tx2gene.html">tx2gene</a></code> </p>
        </td>
        <td><p>Transcript to gene mapping</p></td>
      </tr></tbody></table></div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Fabrício Almeida-Silva, Thiago Venancio.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.3.</p>
</div>

      </footer></div>

  


  

  </body></html>

