<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Function reference • bears</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.1.3/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.1.3/bootstrap.bundle.min.js"></script><link href="../deps/Atkinson_Hyperlegible-0.4.2/font.css" rel="stylesheet"><link href="../deps/Source_Code_Pro-0.4.2/font.css" rel="stylesheet"><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><meta property="og:image" content="/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-#0092ac"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">bears</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.99.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/bears_vignette.html">bears: building expression atlases in R</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/almeidasilvaf/bears/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-index">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>Function reference</h1>
    </div>

    <div class="section level2">
      <h2 id="all-functions">All functions<a class="anchor" aria-label="anchor" href="#all-functions"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="check_md5.html">check_md5()</a></code> 
        </dt>
        <dd>Check file integrity with md5sum</dd>
      </dl><dl><dt>
          
          <code><a href="create_dir_structure.html">create_dir_structure()</a></code> 
        </dt>
        <dd>Create a standard directory for bears</dd>
      </dl><dl><dt>
          
          <code><a href="create_sample_info.html">create_sample_info()</a></code> 
        </dt>
        <dd>Search the SRA database and create a data frame of sample metadata</dd>
      </dl><dl><dt>
          
          <code><a href="download_from_ena.html">download_from_ena()</a></code> 
        </dt>
        <dd>Download FASTQ files from ENA's FTP</dd>
      </dl><dl><dt>
          
          <code><a href="fastq_exists.html">fastq_exists()</a></code> 
        </dt>
        <dd>Check if FASTQ files were properly downloaded</dd>
      </dl><dl><dt>
          
          <code><a href="fastqc_is_installed.html">fastqc_is_installed()</a></code> 
        </dt>
        <dd>Check if FastQC is installed</dd>
      </dl><dl><dt>
          
          <code><a href="fastqc_table.html">fastqc_table</a></code> 
        </dt>
        <dd>FastQC summary table</dd>
      </dl><dl><dt>
          
          <code><a href="fcount.html">fcount()</a></code> 
        </dt>
        <dd>Count reads with featureCounts</dd>
      </dl><dl><dt>
          
          <code><a href="featureCounts2se.html">featureCounts2se()</a></code> 
        </dt>
        <dd>Create a SummarizedExperiment object from featureCounts output</dd>
      </dl><dl><dt>
          
          <code><a href="get_url_ena.html">get_url_ena()</a></code> 
        </dt>
        <dd>Get URL for each file in the ENA's FTP repository</dd>
      </dl><dl><dt>
          
          <code><a href="gff2bed.html">gff2bed()</a></code> 
        </dt>
        <dd>Convert GFF file to BED</dd>
      </dl><dl><dt>
          
          <code><a href="infer_strandedness.html">infer_strandedness()</a></code> 
        </dt>
        <dd>Infer library strandedness</dd>
      </dl><dl><dt>
          
          <code><a href="kallisto2se.html">kallisto2se()</a></code> 
        </dt>
        <dd>Create a SummarizedExperiment object from kallisto output</dd>
      </dl><dl><dt>
          
          <code><a href="kallisto_index.html">kallisto_index()</a></code> 
        </dt>
        <dd>Index the transcriptome for kallisto</dd>
      </dl><dl><dt>
          
          <code><a href="kallisto_is_installed.html">kallisto_is_installed()</a></code> 
        </dt>
        <dd>Check if kallisto is installed</dd>
      </dl><dl><dt>
          
          <code><a href="kallisto_quantify.html">kallisto_quantify()</a></code> 
        </dt>
        <dd>Quantify expression with kallisto</dd>
      </dl><dl><dt>
          
          <code><a href="mapping_pass.html">mapping_pass()</a></code> 
        </dt>
        <dd>Keep only samples that passed minimum requirements in STAR alignment</dd>
      </dl><dl><dt>
          
          <code><a href="mapping_qc.html">mapping_qc</a></code> 
        </dt>
        <dd>Summary statistics of STAR mapping QC</dd>
      </dl><dl><dt>
          
          <code><a href="multiqc.html">multiqc()</a></code> 
        </dt>
        <dd>Run MultiQC to get a summary of QC results</dd>
      </dl><dl><dt>
          
          <code><a href="multiqc_is_installed.html">multiqc_is_installed()</a></code> 
        </dt>
        <dd>Check if multiqc is installed</dd>
      </dl><dl><dt>
          
          <code><a href="remove_rrna.html">remove_rrna()</a></code> 
        </dt>
        <dd>Remove rRNA sequences from .fastq files with SortMeRNA</dd>
      </dl><dl><dt>
          
          <code><a href="rseqc_is_installed.html">rseqc_is_installed()</a></code> 
        </dt>
        <dd>Check if RSeQC is installed</dd>
      </dl><dl><dt>
          
          <code><a href="run_fastqc.html">run_fastqc()</a></code> 
        </dt>
        <dd>Run FastQC on .fastq files</dd>
      </dl><dl><dt>
          
          <code><a href="salmon2se.html">salmon2se()</a></code> 
        </dt>
        <dd>Create a SummarizedExperiment object from salmon output</dd>
      </dl><dl><dt>
          
          <code><a href="salmon_index.html">salmon_index()</a></code> 
        </dt>
        <dd>Index the transcriptome for salmon</dd>
      </dl><dl><dt>
          
          <code><a href="salmon_is_installed.html">salmon_is_installed()</a></code> 
        </dt>
        <dd>Check if salmon is installed</dd>
      </dl><dl><dt>
          
          <code><a href="salmon_quantify.html">salmon_quantify()</a></code> 
        </dt>
        <dd>Quantify expression with salmon</dd>
      </dl><dl><dt>
          
          <code><a href="sample_info.html">sample_info</a></code> 
        </dt>
        <dd>Sample metadata</dd>
      </dl><dl><dt>
          
          <code><a href="sortmerna_is_installed.html">sortmerna_is_installed()</a></code> 
        </dt>
        <dd>Check if SortMeRNA is installed</dd>
      </dl><dl><dt>
          
          <code><a href="sratoolkit_is_installed.html">sratoolkit_is_installed()</a></code> 
        </dt>
        <dd>Check if SRAToolkit is installed</dd>
      </dl><dl><dt>
          
          <code><a href="star_align.html">star_align()</a></code> 
        </dt>
        <dd>Align reads to a reference genome using STAR</dd>
      </dl><dl><dt>
          
          <code><a href="star_genome_index.html">star_genome_index()</a></code> 
        </dt>
        <dd>Index genome for STAR alignment</dd>
      </dl><dl><dt>
          
          <code><a href="star_is_installed.html">star_is_installed()</a></code> 
        </dt>
        <dd>Check if STAR is installed</dd>
      </dl><dl><dt>
          
          <code><a href="stringtie2se.html">stringtie2se()</a></code> 
        </dt>
        <dd>Create a SummarizedExperiment object from StringTie output</dd>
      </dl><dl><dt>
          
          <code><a href="stringtie_assemble.html">stringtie_assemble()</a></code> 
        </dt>
        <dd>Assemble transcripts with StringTie</dd>
      </dl><dl><dt>
          
          <code><a href="stringtie_is_installed.html">stringtie_is_installed()</a></code> 
        </dt>
        <dd>Check if StringTie is installed</dd>
      </dl><dl><dt>
          
          <code><a href="stringtie_quantify.html">stringtie_quantify()</a></code> 
        </dt>
        <dd>Quantify expression in TPM with StringTie</dd>
      </dl><dl><dt>
          
          <code><a href="subread_is_installed.html">subread_is_installed()</a></code> 
        </dt>
        <dd>Check if subread is installed</dd>
      </dl><dl><dt>
          
          <code><a href="taco_is_installed.html">taco_is_installed()</a></code> 
        </dt>
        <dd>Check if TACO is installed</dd>
      </dl><dl><dt>
          
          <code><a href="taco_merge.html">taco_merge()</a></code> 
        </dt>
        <dd>Merge assembled transcripts with TACO</dd>
      </dl><dl><dt>
          
          <code><a href="translate_strandedness.html">translate_strandedness()</a></code> 
        </dt>
        <dd>Translate library orientation terminology for each program</dd>
      </dl><dl><dt>
          
          <code><a href="trim_reads.html">trim_reads()</a></code> 
        </dt>
        <dd>Run Trimmomatic on reads that failed on FastQC check</dd>
      </dl><dl><dt>
          
          <code><a href="trimmomatic_is_installed.html">trimmomatic_is_installed()</a></code> 
        </dt>
        <dd>Check if Trimmomatic is installed</dd>
      </dl><dl><dt>
          
          <code><a href="tx2gene.html">tx2gene</a></code> 
        </dt>
        <dd>Transcript to gene mapping</dd>
      </dl></div>
  </main></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Fabrício Almeida-Silva, Thiago Venancio.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

    </footer></div>

  

  

  </body></html>

